docs/site/home.html

```diff
@@ -96,26 +96,35 @@ <h3 class="block-title">What is it?</h3>
         </div>
 
         <div hljs language="js" ng-show="selected == 'compute engine'">
-// Auth is handled automatically inside GCE.
-var gcloud = require('gcloud');
-var dataset = gcloud.datastore.dataset();
-var productKey = dataset.key(['Product', 123]);
+// Authentication is automatic
+// inside Google Compute Engine.
+var gcloud = require('gcloud')({
+  projectId: 'grape-spaceship-123'
+});
+
+var gcs = gcloud.storage();
 
-dataset.get(productKey, function(err, entity) {
-  console.log(err, entity);
+var backups = gcs.bucket('backups');
+backups.upload('db.zip', function(err, file) {
+  // file.createReadStream();
+  // file.getMetadata();
+  // ...
 });</div>
 
         <div hljs language="js" ng-show="selected == 'elsewhere'">
 var gcloud = require('gcloud')({
-  projectId: 'my-project',
+  projectId: 'grape-spaceship-123',
+  // Specify a path to a keyfile.
   keyFilename: '/path/to/keyfile.json'
 });
 
-var dataset = gcloud.datastore.dataset();
-var productKey = dataset.key(['Product', 123]);
+var gcs = gcloud.storage();
```

Comment:
How about we err on consistency's side now, and discuss a big sweeping "change all var storage -> var gcs" change as a separate issue?